packages/components/collapsible-panel/src/collapsible-panel.js

```diff
@@ -211,6 +218,27 @@ CollapsiblePanel.propTypes = {
     // uncontrolled component does not have `onToggle` so no validation needed.
     return null;
   },
+  /**
+   * Horizontal size limit of the input fields.
+   */
+  horizontalConstraint: PropTypes.oneOf([
+    'm',
+    'l',
+    'xl',
```

Comment:
I would leave them out, as this is a new prop and we don't need to be backwards compatible.